src/test/java/data/TaskManagerTest.java

```diff
@@ -291,6 +291,79 @@ void addEvent_invalidInput_addsTask() {
         assertEquals("Invalid task type given. T for Todo, E for event, D for deadline.", thrown.getMessage());
     }
 
+    @Test
+    void updateEventDescriptionOnly_validInput_updatesTask() throws TaskManagerException {
```

Comment:
Can we use "_" while naming methods?

Reply:
Yup, as per coding conventions 
![image](https://github.com/AY2324S2-CS2113-W13-2/tp/assets/110753446/2327bbf2-0f42-42ad-8fb6-587bc46ab985)